src/app/vault/ciphers.component.ts

```diff
@@ -37,6 +38,11 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
   actionPromise: Promise<any>;
   userHasPremiumAccess = false;
 
+  organizationNames = new Map<string, string>();
+  sortedDescending = true;
```

Comment:
The default sort order should be ascending, that is, A to Z. (I had to look this up because ascending/descending is confusing 😂 )